runelite-client/src/test/java/net/runelite/client/plugins/chatfilter/ChatFilterPluginTest.java

```diff
@@ -382,4 +382,16 @@ public void publicChatFilteredOnDuplicate()
 
 		assertEquals(0, client.getIntStack()[client.getIntStackSize() - 3]);
 	}
+
+	@Test
+	public void testDuplicateChatFilterIgnoresColor()
```

Comment:
This test appears unrelated to the issue, and also passes on master without any modifications to the plugin.

Reply:
My bad, published the wrong version.